app/assets/javascripts/admin/addon/templates/components/watched-word-form.hbs

```diff
@@ -17,6 +17,13 @@
   </div>
 {{/if}}
 
+{{#if canLink}}
+  <div class="watched-word-input">
+    <label for="watched-replacement">{{i18n "admin.watched_words.form.link_label"}}</label>
+    {{text-field id="watched-replacement" value=replacement disabled=formSubmitted class="watched-word-input-field" autocorrect="off" autocapitalize="off" placeholderKey="admin.watched_words.form.link_placeholder"}}
```

Comment:
I think the id needs to be unique within a HTML document so we can't just reuse the id from the input above.

Reply:
The two inputs will never show at the same time, so the ID is unique.

Reply:
ahh icic got it :+1: